akka-actor/src/main/java/akka/annotation/ApiMayChange.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.annotation;
+
+import java.lang.annotation.*;
+
+/**
+ * Marks APIs that are meant to evolve towards becoming stable APIs, but are not stable APIs yet.
+ * 
+ * Evolving interfaces can change from one feature release to another release (i.e. 2.1 to 2.2) without up-front notice.
```

Comment:
Ah, it seems that this looser binary compat policy was actually a case in the `epoch.major.minor` era :)